apps/passport-server/src/util/telegramHelpers.ts

```diff
@@ -73,6 +78,70 @@ function isFulfilled<T>(
   return promiseSettledResult.status === "fulfilled";
 }
 
+export const setBotInfo = async (
+  bot: Bot<BotContext, Api<RawApi>>
+): Promise<void> => {
+  if (process.env.PASSPORT_CLIENT_URL) {
```

Comment:
Sets the menu button to be Zupass

![IMG_34CFC4D813AB-1](https://github.com/proofcarryingdata/zupass/assets/42984734/f16b3b7b-f385-4c29-9f50-cea506eee56b)